.circleci/config.yml

```diff
@@ -19,6 +19,11 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: install dependencies
+          command: |
+            gem install bundler -v 2.0.1
```

Comment:
no wonder I couldn't track this down - this direction would have been like, plan D